### DISCUSSION TITLE
Missing VVP documentation?

### DISCUSSION THREAD

**Discussion by martinwhitaker** [steveicarus/iverilog#1064] [General]
There used to be a README.txt in the vvp subdirectory that contained a lot of useful information about the vvp implementation. It has gone, and I haven't found the same information in the Documentation subdirectory.

**Comment by larsclausen** [comment]
Its in https://github.com/steveicarus/iverilog/blob/master/Documentation/developer/guide/vvp/vvp.rst

**Comment by martinwhitaker** [comment]
Thanks.
